packages/node_modules/@webex/plugin-meetings/src/constants.js

```diff
@@ -64,6 +65,39 @@ export const MEETING_STATE_MACHINE = {
     ERROR: 'ERROR'
   }
 };
+export const EVENT_TRIGGERS = {
+  MEDIA_AUDIO: 'media:audio',
```

Comment:
Can we either add a `:` to the key here or change the value to match? It will be incongruent with other things that use underscores.